pvactools/tools/pvacview/anchor_and_helper_functions.R

```diff
@@ -75,10 +82,29 @@ peptide_coloring <- function(hla_allele, peptide_row) {
     return(c("#999999"))
   }
   position <- as.numeric(peptide_row["x_pos"])
-  anchor_score <- as.numeric(anchor_data[[peptide_length]][anchor_data[[peptide_length]]["HLA"] == hla_allele][2:(peptide_length + 1)])
-  value_bins <- cut(anchor_score, breaks = seq(0, 1, len = 100),
-                    include.lowest = TRUE)
-  colors <- colorRampPalette(c("lightblue", "blue"))(99)[value_bins]
+  if (any(hla_allele == anchor_data[[peptide_length]]["HLA"])) {
+    anchor_score <- as.numeric(anchor_data[[peptide_length]][anchor_data[[peptide_length]]["HLA"] == hla_allele][2:(peptide_length + 1)])
+    value_bins <- cut(anchor_score, breaks = seq(0, 1, len = 100),
+                      include.lowest = TRUE)
+    colors <- colorRampPalette(c("lightblue", "blue"))(99)[value_bins]
+  } else if (any(hla_allele == mouse_anchor_data[[peptide_length]]["Allele"])) {
+    mouse_position_data <- (mouse_anchor_data[[peptide_length]][mouse_anchor_data[[peptide_length]]["Allele"] == hla_allele][2:(peptide_length + 1)])
+    colors <- list()
+    for (i in 1:length(mouse_position_data)) {
+      if (mouse_position_data[i] == "True") {
+        colors <- append(colors, "blue")
+      } else {
+        colors <- append(colors, "lightblue")
+      }
+    }
+  } else {
+    if ((position == 1) || (position == 2) || (position == peptide_length-1) || (position == peptide_length)) {
```

Comment:
This could probably be simplified to something like:
```suggestion
    if (position %in% c(1, 2, peptide_length-1, peptide_length)) {
```